requirements.txt

```diff
@@ -1,2 +1,4 @@
+git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```

Comment:
@irio: Any ideas? If the silence remains (_drama_ 👑 ) I' upload it to PyPI next week hahaha…